docs/system_overview/process_model.md

```diff
@@ -1,29 +1,36 @@
-#
+# iRODS Process Model
 
-iRODS 4.2+ launches a long-running server process (`/usr/sbin/irodsServer`), which immediately forks the Agent Factory and spawns the Delay Server (`/usr/sbin/irodsDelayServer`).  All incoming API requests are serviced by the Agent Factory, and for every new connection the Agent Factory spawns a new Agent.  Each Agent will service its request as quickly as possible, and then terminate.
+When iRODS starts, it launches a long-running server process:  
+`/usr/sbin/irodsServer`
 
-The iRODS Delay Server sleeps most of the time, but spawns an Agent every 30 seconds to check the delay queue for any delayed rules that need to be run.
+This process immediately creates two other processes:
 
-Agent processes are listed with the IP address of the connecting client.
+- **Agent Factory**: Handles all incoming API requests
+- **Delay Server** (optional): Manages delayed rule execution
 
-    /usr/sbin/irodsServer
-    ├─ irodsDelayServer
-    └─ irodsAgentFactory
-       ├─ irodsAgent
-       ├─ irodsAgent
-       ├─ irodsAgent
-       └─ irodsAgent
+## How requests are handled
 
-| iRODS Process         | Expected Duration        | Expected Resident Memory Usage  |
-| --------------------- | ------------------------ | ------------------------------- |
-| irodsServer           | long-running             | 126M                            |
-| irodsDelayServer      | long-running             | 17M                             |
-| irodsAgentFactory     | long-running             | 5M                              |
-| irodsAgent            | single client connection | 20M to 100M                     |
+The **Agent Factory** listens for new client connections. For each new connection, it forks a separate **Agent** process. Each Agent handles one client request, completes the work as quickly as possible, and then waits for the next request. Agents will continue to service requests until the client disconnects.
 
-iRODS uses a dynamic linking model, rather than a static linking model, which adds significant overhead for short running processes (e.g. `ils`).  In order to mitigate this overhead, the irodsServer no longer calls `fork()` and `exec()` as a separate executable (pre-4.2).  In addition, since calling `fork()` within a threaded environment creates undefined behavior, the irodsServer calls `fork()` to spawn the Agent Factory process early, before any threading begins and any configuration is determined.
+## The Delay Server
 
-![iRODS Process Model Diagram](../images/process_model_diagram.jpg)
+The **Delay Server** is designed to remain idle most of the time. Every 30 seconds _(by default)_, it wakes up and spawns an Agent to check for any delayed rules that need to run. You can adjust this interval using the `delay_server_sleep_time_in_seconds` property in server_config.json. See [Configuration](../system_overview/configuration.md#etcirodsserver_configjson) for more information about this property.
```

Comment:
does it spawn an agent?  or does it make a client request which causes the agent factory to spawn an agent?

Reply:
I think `spawns` is doing a lot of work in that sentence.

> or does it make a client request which causes the agent factory to spawn an agent?

Yes.

Reply:
right, let's be targeted/specific in our words.   feels like it will help us in the long term.

Reply:
Done.